tests/framework/extensions/nodes/nodes.go

```diff
@@ -0,0 +1,123 @@
+package nodes
+
+import (
+	"github.com/rancher/norman/types"
+	v3 "github.com/rancher/rancher/tests/framework/clients/rancher/generated/management/v3"
+	"github.com/rancher/rancher/tests/framework/clients/rancher"
+	"github.com/rancher/rancher/tests/framework/extensions/kubeapi/secrets"
+	corev1 "k8s.io/api/core/v1"
+	"github.com/sirupsen/logrus"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+const (
+	initNodeLabelKey         = "rke.cattle.io/init-node"
+	initNodeLabelValue       = "true"
+	local                    = "local"
+	machinePlanSecretType    = "rke.cattle.io/machine-plan"
+	machineNameSteveLabel    = "rke.cattle.io/machine-name"
+	node                     = "node"
+)
+
+// Accepts a client and clusterID and returns the init node object
+func GetInitNode(client *rancher.Client, clusterID string) (*v3.Node, error) {
```

Comment:
nit; in the function descriptions, can you have it say `GetInitNode accepts a client...`? Our godocs typically stylize functions in this way.

This same comment goes for all of the functions.